source/includes/connection-snippets/x509-snippets/c-connection.c

```diff
@@ -0,0 +1,32 @@
+// begin x509 connection
+#include <mongoc/mongoc.h>
+
+int
+main (int argc, char *argv[])
+{
+   mongoc_collection_t *collection;
+   mongoc_client_t *client;
+   int64_t count;
+
+   mongoc_init ();
+
+   const mongoc_ssl_opt_t *ssl_default = mongoc_ssl_opt_get_default ();
```

Comment:
In https://github.com/mongodb/docs-ecosystem/pull/573/files#r348473040 and https://github.com/mongodb/docs-ecosystem/pull/573/files#r348478333 we talked about removing `mongoc_ssl_opt_t` entirely and relying on the `tls*` options. This would make the C example more consistent with other languages, since we can rely on the connection string alone.

Reply:
Fixed